icu4c/source/i18n/number_patternmodifier.h

```diff
@@ -196,22 +196,22 @@ class U_I18N_API MutablePatternModifier
     const bool fStrong;
 
     // Pattern details (initialized in setPatternInfo and setPatternAttributes)
-    const AffixPatternProvider *patternInfo;
-    UNumberSignDisplay signDisplay;
+    const AffixPatternProvider *mPatternInfo;
```

Comment:
Again, we usually use `f` for member variables

Reply:
Thanks Shane.

---
In reply to: [208066709](https://github.com/unicode-org/icu/pull/18#discussion_r208066709) [](ancestors = 208066709)